05a-run_ica.py

```diff
@@ -44,10 +44,9 @@ def run_ica(subject, tsss=config.mf_st_duration):
         events_list.append(events)
 
         # mark bads from any run
-        if run:
-            bads = set(sum(config.bads[subject].values(), []))
-        else:
-            bads = config.bads[subject]
+        bads = config.bads[subject]
+        if isinstance(bads, dict):
+            bads = set(sum(bads.values(), []))
```

Comment:
@Alex, I don't understand what 
`bads = set(sum(bads.values(), []))`
does

Reply:
this creates a set that contains the union of all bads across runs